.babelrc

```diff
@@ -3,9 +3,6 @@
   "loose": true,
   "sourceMaps": "inline",
   "blacklist" : [ "strict" ],
-  "optional"  : [
-    "minification.inlineExpressions",
-    "minification.propertyLiterals",
-    "minification.memberExpressionLiterals"
-  ]
+  "optional"  : [],
```

Comment:
These optimizations yielded little benefit (if any) and threw a bunch of babel warnings